src/runtime_src/core/edge/drm/zocl/common/cu.c

```diff
@@ -445,18 +444,22 @@ static int cu_probe(struct platform_device *pdev)
 	kfree(zcu);
 	return err;
 }
-
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 12, 0)
+static void cu_remove(struct platform_device *pdev)
```

Comment:
why? Lets not change the return type. Instead of that, return proper result in this function.
Am i missing anything?

Reply:
Hi Vamshi, In kernel version 6.12 and newer, the return type is void, whereas in the older kernels like 6.6 it's int. To maintain backward compatibility, I have added a macro.

Reply:
Got it. Thanks Manoj